packages/grafana-prometheus/src/index.ts

```diff
@@ -85,3 +85,11 @@ export {
   type StandardPromVariableQuery,
 } from './types';
 export { PrometheusVariableSupport } from './variables';
+
+// For explore metrics
+export { default } from './language_provider';
```

Comment:
It feels a little scary adding a new default export to `@grafana/prometheus` (especially because changing this later feels like a potentially breaking change)...perhaps we could do this instead?

```suggestion
export { default as PromQlLanguageProvider } from './language_provider';
```

I tried this locally and updated to a named import in `public/app/features/trails/helpers/MetricDatasourceHelper.ts`, and it seemed to work nicely.